cfme/tests/middleware/test_middleware_deployment.py

```diff
@@ -132,3 +133,228 @@ def test_tags(provider):
+    deployment.undeploy()
+    check_deployment_disabled(provider, server, runtime_name)
+    deployment = get_deployment_from_list(provider, server, runtime_name)
+    deployment.validate_properties()
+
+
+# disable for now as not stable test because of performance issue #10138
+@pytest.mark.uncollect
+@pytest.mark.parametrize("archive", [RESOURCE_WAR_PATH, RESOURCE_JAR_PATH, RESOURCE_EAR_PATH])
+def test_redeploy(provider, archive):
+    """Tests Redeployment of undeployed archive into EAP7 server
+
+    Steps:
+        * Get servers list from UI
+        * Chooses JBoss EAP server from list
+        * Deploys some deployment archive into server.
+        * Undeploys that archive.
+        * Refreshes the provider.
+        * Lists all deployments on EAP server.
+        * Verified the recently redeployed archive's status is Enabled.
+        * Selects that archive from the list to load details.
+        * Verifies that properties of deplyment's summary page and the status is Enabled.
+    """
+    server = get_eap_server(provider)
+    runtime_name = deploy(provider, server, archive)
+    check_deployment_enabled(provider, server, runtime_name)
+    deployment = get_deployment_from_list(provider, server, runtime_name)
+    deployment.undeploy()
+    check_deployment_disabled(provider, server, runtime_name)
+    deployment.redeploy()
+    # enable when #9876 is fixed
+    # _check_deployment_enabled(provider, server, runtime_name)
+    # deployment = _get_deployment_from_list(provider, server, runtime_name)
+    # deployment.validate_properties()
+
+
+# disable for now as not stable test because of performance issue #10138
+@pytest.mark.uncollect
+@pytest.mark.parametrize("archive", [RESOURCE_WAR_PATH, RESOURCE_JAR_PATH, RESOURCE_EAR_PATH])
+def test_stop(provider, archive):
+    """Tests Stopping of archive from EAP7 server
+
+    Steps:
+        * Get servers list from UI
+        * Chooses JBoss EAP server from list
+        * Deploys some deployment archive into server.
+        * Select that deployment from deployments list.
+        * Performs "Stop" toolbar operation on it.
+        * Refreshes the provider.
+        * Lists all deployments on EAP server.
+        * Verified the recently stopped archive's status is Disabled.
+        * Selects that archive from the list to load details.
+        * Verifies that properties of deplyment's summary page and the status is Disabled.
+    """
+    server = get_eap_server(provider)
+    runtime_name = deploy(provider, server, archive)
+    check_deployment_enabled(provider, server, runtime_name)
+    deployment = get_deployment_from_list(provider, server, runtime_name)
+    deployment.stop()
+    check_deployment_disabled(provider, server, runtime_name)
+    deployment = get_deployment_from_list(provider, server, runtime_name)
+    deployment.validate_properties()
+
+
+# disable for now as not stable test because of performance issue #10138
```

Comment:
If this is a github issue, you can do:
```python
@pytest.mark.meta(blockers=['GH#owner/repo:10138'])
```

to skip the test with a message. GitHub skipping only works on upstream so far but you do run on upstream :)